test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/TestDescription.java

```diff
@@ -71,7 +71,7 @@
  *      newclass
  *
  * @run main/othervm/native
- *      -agentlib:hs201t002=pathToNewByteCode=./bin,-waittime=5
- *      nsk.jvmti.scenarios.hotswap.HS201.hs201t002
+ *      -agentlib:hs201t002=pathToNewByteCode=./bin,-waittime=5,-verbose
+ *      nsk.jvmti.scenarios.hotswap.HS201.hs201t002 -verbose
```

Comment:
Is this second -verbose a mistake?

Reply:
No. This is the way nsk framework works.
1st -verbose turns on verbose output for agent (native), 2nd one for java part of the test.

Reply:
aha, I didn't see where it was being recognised, thanks!